modules/ROOT/pages/tuning-backend-server.adoc

```diff
@@ -0,0 +1,19 @@
+= Backend Server Response Time
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+When investigating performance issues, you should first look if the overall flow's performance is heavily constrained by the latency (response time or message round-trip time in milliseconds unless otherwise noted) of the remote hosts to which the flow connects into.
+
+Failing to acknowledge the performance overhead that is added to a flow by the underlying complexities of the communication between the application and a remote host, may lead the optimization effort through the wrong path with negligible improvements in the end. +
+Therefore, it is important that before starting any optimization in the Mule app, to first characterize the backend servers average latency and throughput, and to establish if these are the ones limiting the scalability or performance of your application.
```

Comment:
SUGGEST REPLACING because it is repetitive, but:
```suggestion
Therefore, it is important that, before you begin any optimization in the Mule app, you first characterize the backend servers' average latency and throughput to determine if this is what is limiting your application's scalability or performance.
```